File: seed/java-sdk/exhaustive/local-files/src/main/java/com/snippets/Example32.java
package com.snippets;

import com.fern.sdk.SeedExhaustiveClient;
import com.fern.sdk.resources.endpoints.pagination.requests.ListItemsRequest;

public class Example32 {
    public static void main(String[] args) {
        SeedExhaustiveClient client = SeedExhaustiveClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.endpoints().pagination().listItems(
            ListItemsRequest
                .builder()
                .cursor("cursor")
                .limit(1)
                .build()
        );
    }
}

File: seed/csharp-sdk/exhaustive/explicit-namespaces/src/SeedApi.DynamicSnippets/Example32.cs
using SeedExhaustive;
using SeedExhaustive.Endpoints.Pagination;

namespace Usage;

public class Example32
{
    public async Task Do() {
        var client = new SeedExhaustiveClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Pagination.ListItemsAsync(
            new ListItemsRequest {
                Cursor = "cursor",
                Limit = 1
            }
        );
    }

}
